concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1695,120 +1751,25 @@ jobs:
-            - name: paas-cf
-            - name: cf-secrets
-            - name: cf-manifest
-          outputs:
-            - name: config
-          params:
-          image_resource:
-            type: docker-image
-            source:
-              repository: ruby
-              tag: 2.2-slim
-          run:
-            path: sh
-            args:
-              - -e
-              - -c
-              - |
-                VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-
-                CF_ADMIN=admin
-                CF_PASS=$($VAL_FROM_YAML secrets.uaa_admin_password cf-secrets/cf-secrets.yml)
-                API_ENDPOINT=$($VAL_FROM_YAML properties.cc.srv_api_uri cf-manifest/cf-manifest.yml)
-
-                for var_name in CF_ADMIN CF_PASS API_ENDPOINT; do
-                  echo export "${var_name}"=\"$(eval echo \$${var_name})\"
-                done > config/config.sh
-
-      - task: deploy-healthcheck
-        config:
-          platform: linux
-          image_resource:
-            type: docker-image
-            source:
-              repository: governmentpaas/cf-cli
-          params:
-            DISABLE_HEALTHCHECK_DB: {{disable_healthcheck_db}}
-          inputs:
-            - name: paas-cf
-            - name: config
-            - name: bosh-CA
-          outputs:
-            - name: deployed-healthcheck
-          run:
-            path: sh
-            args:
-              - -e
-              - -u
-              - -c
-              - |
-                ./paas-cf/concourse/scripts/import_bosh_ca.sh
-
-                . ./config/config.sh
-                echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-                cf create-org testers
-                cf set-quota testers test_apps
-                cf create-space healthcheck -o testers
-                cf target -o testers -s healthcheck
-                BUILD_ROOT=$(pwd)
-                cd paas-cf/platform-tests/example-apps/healthcheck
-                cf push --no-start
-                if  [ "${DISABLE_HEALTHCHECK_DB:-}" != "true" ] ; then
-                  cf create-service postgres Free healthcheck-db
-                  while ! cf service healthcheck-db | grep -q 'Status: create succeeded'; do
-                    echo "Waiting for creation of service to complete..."
-                    sleep 30
-                  done
-                  cf bind-service healthcheck healthcheck-db
-                fi
-
-                cf start healthcheck
-                cd $BUILD_ROOT
-                echo "yes" > deployed-healthcheck/healthcheck-deployed
-        on_success:
-          put: deployed-healthcheck
-          params:
-            file: deployed-healthcheck/healthcheck-deployed
-
   - name: smoke-tests
     serial_groups: [smoke-tests]
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: ['post-deploy','availability-tests']
+            passed: ['availability-tests']
```

Comment:
This needs to trigger from both `cf-deploy` and `availability-tests`. Otherwise if the deploy fails, but the availability tests are green, the pipeline will continue, which we don't want. (similarly for `paas-cf` below)